benchmark-service/benchmark/benchmark.py

```diff
@@ -240,15 +316,23 @@ async def update_commits(app):
 
 
 async def get_commit(app, sha):  # pylint: disable=unused-argument
+    log.info('in get_commit')
     github_client = app['github_client']
     batch_client = app['batch_client']
     gs_reader = app['gs_reader']
 
-    file_path = f'{BENCHMARK_RESULTS_PATH}/{sha}.json'
+    file_path = f'{BENCHMARK_RESULTS_PATH}/0-{sha}.json'
     request_string = f'/repos/hail-is/hail/commits/{sha}'
-
     gh_commit = await github_client.getitem(request_string)
 
+    message = gh_commit['commit']['message']
+    idx = message.index('#')
+    commit_id = message[idx: idx + 5]
+
+    title_message = gh_commit['commit']['message']
+    title_end = title_message.index(')')
```

Comment:
Can you explain to me what problem you are solving with this?

Reply:
When I was just getting the title, it would give me a string with the title and the pr number and all the messages on the pr, so I was trying to get just the title by getting the substring until the pr number

Reply:
hmmm... I'm thinking a regex for this and the PR number above might be better. It will at least be easier to understand.